Marlin/src/sd/usb_flashdrive/lib/version_helper.h

```diff
@@ -27,175 +27,177 @@ e-mail   :  support@circuitsathome.com
  * Includes fixes for versions as low as 0023, used by Digilent.
  */
 
-#if defined(ARDUINO) && ARDUINO >=100
 #include <Arduino.h>
```

Comment:
@ThinkyHead: The code compiles just fine without this file. I initially had removed it, but then decided to keep it since it was in upstream and I did not know whether it was necessary. If you don't think it is needed, we can remove it.